scapy-ssl_tls/ssl_tls.py

```diff
@@ -8,7 +8,7 @@
 import os, time
 
 try:
-    import ssl_tls_crypto
+    from scapy_ssl_tls import ssl_tls_crypto
```

Comment:
This line resoles the following errors for me:

Import Error - most likely due to missing pycrypto libraries - disabling crypto functionality
ImportError('cannot import name TLSHandshake',).  Perhaps it can be split from this PR?

:+1: